packages/google-cloud-ids/CONTRIBUTING.rst

```diff
@@ -35,21 +35,21 @@ Using a Development Checkout
 You'll have to create a development environment using a Git checkout:
 
 - While logged into your GitHub account, navigate to the
-  ``python-ids`` `repo`_ on GitHub.
+  ``google-cloud-python`` `repo`_ on GitHub.
```

Comment:
These are the updates we'll have to do for all migrated repos. Not sure we can automate this...

Reply:
Isn't this just a string replace, so can be done during your post-processing step?